config/oidc-auth.php

```diff
@@ -57,13 +57,18 @@
     | JWT claims in id_token required to authenticate. Arrays set required
     | elements in an array. Other values are matched exactly.
     |
-    | This can also be a Closure to check id_token. id_token will be passed as
-    | the first parameter. Return true to indicate a pass or false for a fail.
+    | You can provide a \Closure or
+    | \Laravel\SerializableClosure\SerializableClosure that accepts id_token as
+    | its argument to perform customized validation.
+    | Return true to indicate a pass or false for a fail.
+    | When passing SerializableClosure, use `APP_KEY` environment variable to sign
+    | the serialized data.
     |
      */
     'required_claims' => [
-        //'name' => 'value',
-        //'array' => ['required', 'elements'],
+        // 'name' => 'value',
+        // 'array' => ['required', 'elements'],
+        // 'closure' => serialize(new SerializableClosure($closure))
```

Comment:
i believe what you implementing is `'required_claims' => serialize(new SeralizableClosure()),`.

we should also take this chance to make it more like realistic, like

```
// 'required_claims' => fn ($claims) => $claims->has('name') && $claims->get('name') === 'value'),`
// 'required_claims' => serialize(new SeralizableClosure(fn ($claims) => $claims->has('name') && $claims->get('name') === 'value')),`
```

below the array example, with proper formatting